paper-toast.html

```diff
@@ -170,9 +170,16 @@
         },
 
         /**
-         * Show the toast. Same as `open()` from `IronOverlayBehavior`.
+         * Show the toast. Without arguments, this is the same as `open()` from `IronOverlayBehavior`.
          */
-        show: function() {
+        show: function(properties) {
+          properties = properties || {};
+          if (typeof properties == 'string') {
+            this.text = properties;
+          } else {
+            var t = this;
+            Object.keys(properties).forEach(function(key){ t[key] = properties[key]; });
```

Comment:
I'm afraid of this blind setting of properties, because it could lead on a mis-usage of the method. 
What if (absurd example) someone does `toast.show({ opened: false })`?
If we want to be more conservative, I would allow only the setting of `text` and `duration` through this method.

If we want to allow all properties to be set, I would at least do some sanity checks e.g. 
```javascript
properties = properties || {};
if (typeof properties === 'string') {
 properties = { text: properties };
}
for (var property in properties) {
  if (property.indexOf('_') === 0) {
    console.warn(property + ' was not set since is private');
  } else if (property in this) {
    this[property] = properties[property];
  } else {
    console.warn(property + ' was not set since it is not a valid property');
  }
}